### ISSUE TITLE
Feature request: Popout the chat window?

### ISSUE THREAD

**Issue by muhfreedomss** [streamlink/streamlink#735]
Is there any way for this program to somehow popout the chat window when you open a stream?
 Like so:

http://i.imgur.com/yRjsNXt.jpg

**Comment by gravyboat** [comment]
Hmm, while this is a possibility it seems outside the scope of Streamlink itself. I'll tag it and we'll see what other people think.

**Comment by bastimeyer** [comment]
I don't think this belongs into Streamlink itself. The [Streamlink Twitch GUI](https://github.com/streamlink/streamlink-twitch-gui) does have a Twitch chat function for opening browser windows or other chat application. So if you want, you can use it instead of the Streamlink CLI.

These minimal browser window layouts you are seeing on your screenshot are being created by already running browser instances and websites which are using a Javascript API for that ([DOM Level 1 window.open](https://developer.mozilla.org/en-US/docs/Web/API/Window/open)). You can't simply launch any browser process like that.

Chromium and Chrome do have a parameter that lets it launch in a new so called app session (`--app="URL"` parameter), but other browsers like Firefox require addons, Edge and Safari don't support minimal window layouts at all and MSIE requires a VBScript launch script. I'm not sure if Opera has support for this and then there are also countless of other browsers and browser forks.

Also figuring out which browser is being used by the user is next to impossible. There are simply too many different setups and configurations.

If you want to open a Twitch chat window while using the Streamlink CLI, you can write a simple shell wrapper script that receives a Twitch channel name as parameter and launches Chromium/Chrome and Streamlink.

**twitch.sh**
```bash
#!/usr/bin/env bash

chromium "--app=https://twitch.tv/${1}/chat" 2>&1 >/dev/null &
streamlink ${@:2} "twitch.tv/${1}" best
```
**Example:**
`twitch.sh CHANNEL -p mpv`
